bin/rucio

```
@@ -1033,6 +1037,7 @@ def download(args):
         trace_pattern['taskid'] = args.trace_taskid
     if args.trace_usrdn:
         trace_pattern['usrdn'] = args.trace_usrdn
+    raise_exception = args.dont_raise_exception if args.dont_raise_exception is not None else True
```

raise_exception is much to generic for what it does; The name makes you think that all exceptions will be ignored; but it only acts on two particular exceptions; there are many others which can still be raised under current code. I vote for either making the option name more specific, or ensuring that all exceptions are handled silently.

---

also, never use negative arguments :-)

--raise-exception, default=False

instead of

--dont-raise-exception, default=True

---

Do we want to disable the exceptions by default?

---

not sure.

for me, the important bit is to have the --argument not phrased inverse ;-)

---

From my pov it should be enabled by default. I would suggest a `--deactivate-exceptions` flag.